client/src/App.jsx

```diff
@@ -2,29 +2,51 @@ import { lazy, Suspense } from "react";
 import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
 import { ErrorBoundary } from "react-error-boundary";
 import NavigationBar from "./components/navbar/NavigationBar";
-
 import { ErrorState, LoadingState } from "./components/states";
 
 const Homepage = lazy(() => import("./pages/home/Homepage"));
 const Activity = lazy(() => import("./pages/activity/Activity"));
 const Events = lazy(() => import("./pages/events/Events"));
 const Projects = lazy(() => import("./pages/projects/Projects"));
 const Resources = lazy(() => import("./pages/resources/Resources"));
+const Login = lazy(() => import("./pages/manager/login/Login"));
+
+
+function Client() {
+  return (
+		<>
+		<NavigationBar />
```

Comment:
Make sure to run prettier on this file because a lot of the indents and spacing are off.